<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmOption_5</class>
 <widget class="QWidget" name="frmOption_5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmOption_5</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>每笔开仓价</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>持仓均价</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>以下指定品种或合约开仓委托成交时，将以</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>调整指定个价位作为</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>触发价自动生成平仓条件单，平仓报单价为:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>最新价</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>买卖价</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>调整</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>个价位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注1:以上设置对所有止赢止损的品种有效。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注2:调整价位提示：1个调价单位对应该品种的价格最小变动单位。如：白银延期的1个调整</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>价位为1原，黄金延期的1个调整价位为0.01元</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modify">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_add</sender>
   <signal>clicked()</signal>
   <receiver>frmOption_5</receiver>
   <slot>slotAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_modify</sender>
   <signal>clicked()</signal>
   <receiver>frmOption_5</receiver>
   <slot>slotModify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete</sender>
   <signal>clicked()</signal>
   <receiver>frmOption_5</receiver>
   <slot>slotDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAdd()</slot>
  <slot>slotModify()</slot>
  <slot>slotDelete()</slot>
 </slots>
</ui>
